pkg/callbacks/callbacks.go

```diff
@@ -45,15 +45,24 @@ func getLine(c Callback, output OutputType, tag string) ([]byte, error) {
 		}
 		return []byte(fmt.Sprintf("%T:%s, %s", output, tag, line)), nil
 	case AnalyserJSON:
-		analyserFormat, err := output.GetAnalyserFormat()
+		outputs, err := output.GetAnalyserFormat()
 		if err != nil {
 			return []byte{}, fmt.Errorf("failed to get AnalyserFormat %w", err)
 		}
-		line, err := json.Marshal(analyserFormat)
-		if err != nil {
-			return []byte{}, fmt.Errorf("failed to marshal AnalyserFormat for %s %w", tag, err)
+		newline := []byte("\n")
```

Comment:
Having a function called `getLine()` explicitly return multi-line output seems counter-intuitive. It may be it needs to be renamed and documented, or the flow might need to be changed slightly.